Gemfile

```diff
@@ -5,7 +5,7 @@ source 'http://rubygems.org'
 gemspec
 
 group :development, :test do
-  gem 'rubocop', '~> 0.65', require: false
+  gem 'rubocop', '~> 0.74.0', require: false
```

Comment:
mmh, can 0.79.0 be used?

Reply:
Yes but it needs to change few lines of code. I'll address it in separate PR if you don't mind. WDYT?

Reply:
seems reasonable